plugin/storage/grpc/shared/grpc_client_test.go

```diff
@@ -166,6 +166,20 @@ func TestGRPCClientGetTrace_NoTrace(t *testing.T) {
 	})
 }
 
+func TestGRPCClientGetTrace_StreamErrorTraceNotFound(t *testing.T) {
+	withGRPCClient(func(r *grpcClientTest) {
+		traceClient := new(grpcMocks.SpanReaderPlugin_GetTraceClient)
+		traceClient.On("Recv").Return(nil, spanstore.ErrTraceNotFound)
+		r.spanReader.On("GetTrace", mock.Anything, &storage_v1.GetTraceRequest{
+			TraceID: mockTraceID,
+		}).Return(traceClient, nil)
+
+		s, err := r.client.GetTrace(context.Background(), mockTraceID)
+		assert.Error(t, err)
```

Comment:
that doesn't test the issue, you probably want ErrorEquals method

Reply:
fixed.